catalogue/webapp/services/prismic/fetch/articles.ts

```diff
@@ -2,32 +2,169 @@ import { Story } from '../types/story';
 import { PrismicResultsList, PrismicApiError, Query } from '../types';
 import { prismicGraphQLClient, prismicApiError } from '.';
 import { transformPrismicResponse } from '../transformers';
+import { gql } from 'graphql-request';
+
+export const storiesQuery = gql`
+  query getAllStories(
+    $queryString: String
+    # I have changed the below type to work with the error message I get from Prismic graphql when I don't use it
```

Comment:
Let's make this comment more specific (refer to the param name) and include a copy of the error message.